packages/shared-ux/markdown/impl/markdown.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiMarkdownEditor, EuiMarkdownEditorProps, EuiMarkdownFormat } from '@elastic/eui';
+import React, { useState } from 'react';
+
+export type MarkdownProps = Partial<
+  Omit<EuiMarkdownEditorProps, 'editorId' | 'uiPlugins' | 'markdownFormatProps'>
+> & {
+  /**
+   * @param readOnly is needed to differentiate where markdown is used as a presentation of error messages
+   * This was previous the MarkdownSimple component
+   */
+  readOnly: boolean;
+  markdownContent?: string;
```

Comment:
Why declare a prop and not use the `children` for the content?

```jsx
<Markdown>
  My content in **markdown** format
</Markdown>
```

Reply:
Sounds good!! Both ways should work now - I added it as a prop vs children because of how the Markdown component in kibana-react is used across Kibana. 
For instance in error.tsx 

```<Markdown markdown={message} openLinksInNewTab={true} />```